_inc/client/privacy/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { ModuleToggle } from 'components/module-toggle';
+import { updateSettings } from 'state/settings';
+import { getSettings } from 'state/settings';
```

Comment:
These two lines import selectors from the same place. The linter should throw an error ( cc @oskosk  ) because we should have a single line importing everything needed from a particular bundle of selectors.